indico/modules/events/papers/controllers/display.py

```diff
@@ -43,6 +43,12 @@ class RHSubmitPaper(RHPaperBase):
     PAPER_REQUIRED = False
     ALLOW_LOCKED = True
 
+    def _check_paper_protection(self):
+        if not RHPaperBase._check_paper_protection(self):
+            return False
+        last_revision = self.contribution._paper_last_revision
+        return last_revision is None or last_revision.state == PaperRevisionState.to_be_corrected
```

Comment:
Do we also want to require the last revision to have been submitted by the same user? Or should any author be able to submit a new one once it's in "to be corrected"?

Reply:
@pferreir what do you think?

Reply:
But i guess we shouldn't limit it to the same user unless there is a good reason for that.

Reply:
I think we should not limit it to the same person. Being flexible in this case is probably the best.

Reply:
Think of co-authors working collaboratively on a paper.